recordedit/form.controller.js

```diff
@@ -128,7 +161,8 @@
                     sortOrder: null,
                     page: page,
                     pageLimit: model.rows.length,
-                    rowValues: DataUtils.getRowValuesFromTuples(page.tuples),
+                    removePseudoColumns: true,
```

Comment:
Not sure if this is the best way to modify how `vm.resultsetModel.columnModels` is created since `vm.resultsetModel.columns` is not used for the table of columns anymore.